### PR TITLE
v0.10.0

### LINKED ISSUE #543: Move calling of INIT_NETWORK_ACTIVITY to App
It's currently in HubiiApiHoc. Makes more sense for it to be called at the App level

### LINKED ISSUE #593: The "recipient" field of a transfer should make it easier to transfer between my own accounts
Only an address or a registered "contact" is valid input, but I think all registered wallets (except the "current" one) should be easily selectable in the dropdown or similar. Maybe make them _implicit_ contacts?

### LINKED ISSUE #710: Users want the application window to be opened in the position that it was last closed, so restore size and location on startup
On my Mac (OSX mojave) the core window is always opened with a default location and size.

Maybe electron has support for this built in?

- [ ]  save window pos and size on exit
- [ ] restore saved window pos and size on startup if valid
- [ ] if no saved position, or saved position not valid any more, use default

### LINKED ISSUE #711: Selecting text in the GUI should only be possible where it makes sense.
Static UI elements should not have a style that allows text selection, except some special cases like read-only input boxes and similar. The current implementation breaks with the desktop paradigm of the application.

### LINKED ISSUE #712: Allow collapsible wallet cards
screenshot from 2019-01-26 17-15-04

### LINKED ISSUE #721: use the dynamic eth node host via nahmii-sdk

### LINKED ISSUE #736: Further optimise the UI performance
The size of data coming from both the transaction and receipt API can be large, and can lead to performance overhead as the selectors do the deep comparison to decide if it is necessary to emit `new data` events to the downstream.

If it can just compare the last record of the response of these two APIs instead of the whole data bundle for every data poll, it could avoid the potential performance issue when the wallet has a lot of transactions.

### LINKED ISSUE #737: Show block height when hover the network status

### PR DESCRIPTION
**Pull request by katat**
Closes #736 
Closes #721 
Closes #710 
Closes #711 
Closes #712 
Closes #737 
Closes #593 
Closes #543

### CODE REVIEW

File: src/containers/HubiiApiHoc/selectors.js [hunk 1]
```diff
@@ -1,58 +1,75 @@
...
-const makeSelectTransactions = () => createSelector(
-  selectHubiiApiHocDomain,
-  (hubiiApiHocDomain) => hubiiApiHocDomain.get('transactions') || fromJS([])
+export const createTransactionsSelector = createSelectorCreator(
+  defaultMemoize,
+  (currentArray, previousArray) => {
+    let changed = false;
+    currentArray.keySeq().forEach((address) => {
+      if (!currentArray.getIn([address, 'transactions']) || !previousArray.getIn([address, 'transactions'])) {
+        return;
```
Comment (morfj) [inline src/containers/HubiiApiHoc/selectors.js]: No return value, just undefined?

Comment (katat) [reply src/containers/HubiiApiHoc/selectors.js]: It is a loop through an array to check if the array has been updated or not. The `return` in this line is to skip the check for the current address if it doesn't meet the sanity condition.

File: src/containers/HubiiApiHoc/tests/selectors.test.js [hunk 2]
```diff
@@ -74,28 +74,28 @@ describe('makeSelectTransactionsWithInfo', () => {
     expect(transactionsWithInfoSelector(mockedState)).toEqual(expected);
   });
 
-  it('should mark add tx as loading when prices loading', () => {
+  xit('should mark add tx as loading when prices loading', () => {
```
Comment (morfj) [inline src/containers/HubiiApiHoc/tests/selectors.test.js]: Disabled tests intentional? Dead code?

Comment (katat) [reply src/containers/HubiiApiHoc/tests/selectors.test.js]: Yes, it is intentional. These tests are not necessary anymore after supported the cache for the transactions data. I will remove these tests.

**Pull request merged**